src/components/PokeCard/PokeCard.js

```diff
@@ -0,0 +1,56 @@
+import axios from "axios";
+import React, {useEffect, useState} from "react";
+import './PokeCard.css'
+
+function PokeCard({ url }) {
+    const [pokemonData, setPokemonData] = useState([]);
+
+
+    useEffect(()=>{ async function fetchData() {
+        try {
+            const result = await
+                axios.get(`${url}`);
+            console.log(result.data);
+            setPokemonData(result.data);
+        } catch (e) {
+            console.error(e);
+        }
+    } fetchData()}, [url])
+
+    return (
+        <>
+
+                <div className='pokemon-card'>
+
+             {Object.keys(pokemonData).length > 0 &&
```

Comment:
Hier kan de check dan wat simpeler
```suggestion
             {pokemonData &&
```

Reply:
Dan wordt mijn data niet geladen.